documentation/src/docs/wire-protocol.md

```diff
@@ -11,102 +11,97 @@ You may obtain a copy of the License at
 
 This page describes the proposed Wire Protocol for the Streaming Service. See parent page for description of the service as a whole.
```

Comment:
@tkaitchuck ` See parent page `....... Which page this refers to ??

Reply:
I think it should be changed to a link to http://pravega.io/docs/latest/pravega-concepts/

Reply:
Accepted Suggestion